Sources/iOS/OAuth2WebViewController.swift

```diff
@@ -115,7 +115,7 @@ open class OAuth2WebViewController: UIViewController, WKNavigationDelegate {
 		// create a web view
 		let web = WKWebView()
 		web.translatesAutoresizingMaskIntoConstraints = false
-		web.scrollView.decelerationRate = UIScrollViewDecelerationRateNormal
+		web.scrollView.decelerationRate = UIScrollView.DecelerationRate.normal
```

Comment:
Could you simply use `.normal`?

Reply:
No because it wouldn't be compatible with 4.0 anymore, and Xcode 10 won't be released for a few months yet.